movement/validators/datasets.py

```diff
@@ -307,3 +308,12 @@ def __attrs_post_init__(self):
                 "Setting to 1-based IDs that are unique per frame: \n"
                 f"{self.individual_names}.\n"
             )
+
+        if self.frame_array is None:
+            self.frame_array = np.arange(self.position_array.shape[0]).reshape(
+                -1, 1
+            )
+            log_warning(
+                "Confidence array was not provided."
```

Comment:
```suggestion
                "Confidence array was not provided. \n"
```
or? You wanted the comment to be printed on two lines?

Reply:
It's fine in one line, but there was actually a space missing after the full stop. Fixed as part of [this PR](https://github.com/neuroinformatics-unit/movement/pull/230) - thanks!